core/managementserver/server.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package managementserver implements the user manager APIs
+package managementserver
+
+import (
+	"context"
+
+	pb "github.com/google/keytransparency/core/api/usermanager/v1/usermanager_proto"
+	keypb "github.com/google/keytransparency/core/proto/keymaster_proto"
+	ktpb "github.com/google/keytransparency/core/proto/keytransparency_v1_proto"
+)
+
+// Server implements pb.UserManagerServiceServer
+type Server struct{}
+
+// New creates a new managementserver
+func New() *Server {
```

Comment:
Do you need the indirection here, or could users just do `s := &Server{}` ?

Reply:
This function will be expanding to take arguments in the future :-)
I quite like calling `New` to create objects. Is that not canonical?

Reply:
A New() func is OK, but since Server is exported, a trivial New might get you worse readability than a Server literal (because of field labels vs method params).

This is just my 2c though.